src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/rest/RestSubmissionCommand.groovy

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Just moved. BTW Note that a derived method throwing a `NotImplementedException` is bad smell indicating a design error.